src/gh/components/DF_vizualization/code.py

```diff
@@ -0,0 +1,69 @@
+#! python3
+
+import System
+import typing
+
+import Rhino
+import Rhino.Geometry as rg
+from ghpythonlib.componentbase import executingcomponent as component
+
+import Grasshopper as gh
+from Grasshopper.Kernel import GH_RuntimeMessageLevel as RML
+
+from diffCheck import diffcheck_bindings
+from diffCheck import df_cvt_bindings
+from diffCheck import df_vizualization
+
+
+class Vizualization(component):
+    def RunScript(self,
+                  i_results,
```

Comment:
naming convention `i_result` if it is only an item. Type hinting and docstrings might be great to have it here too.

Reply:
- [X] done